lib/modules/@apostrophecms/ui/src/apos/components/AposFilterMenu.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <AposContextMenu
+    :origin="origin" :tip-alignment="tipAlignment"
+    :button="button"
+  >
+    <div class="apos-filters-menu">
+      <div
+        v-for="(set, key) in filterSets" :key="key"
+        class="apos-filters-menu__set"
+      >
+        <component
+          :is="map[set.field.type]" :field="set.field"
+          :value="set.value" :status="set.status"
+          @input="input($event, key)"
+        />
+      </div>
+    </div>
+  </AposContextMenu>
+</template>
+
+<script>
+import AposHelpers from '../mixins/AposHelpersMixin';
+
+export default {
+  mixins: [AposHelpers],
+  props: {
+    filters: {
+      type: Array,
+      required: true
+    },
+    button: {
+      type: Object,
+      default() {
+        return {
+          label: 'Filter',
+          icon: 'chevron-down-icon',
+          modifiers: ['icon-right'],
+          type: 'outline'
+        };
+      }
+    },
+    tipAlignment: {
+      type: String,
+      default: 'center'
+    },
+    origin: {
+      type: String,
+      default: 'below'
+    }
+  },
+  data() {
+    return {
+      map: {
+        radio: 'AposRadioInput',
+        checkbox: 'AposCheckboxInput'
+      }
+    };
+  },
+  computed: {
+    filterSets() {
```

Comment:
This computed property is new. Since we're getting the filters from above, this formats them into field objects.